src/primitives.js

```diff
@@ -456,42 +808,21 @@ export default {
 				polyhedronGeometry = new InstancedBufferGeometry().setAttribute(
 					'position',
 					new BufferAttribute(new Float32Array([
-						-vertexPosition,
-						-vertexPosition,
-						vertexPosition,
-						vertexPosition,
-						vertexPosition,
-						vertexPosition,
-						-vertexPosition,
-						vertexPosition,
-						-vertexPosition,
-						vertexPosition,
-						-vertexPosition,
-						-vertexPosition,
-						-vertexPosition,
-						vertexPosition,
-						-vertexPosition,
-						vertexPosition,
-						vertexPosition,
-						vertexPosition,
-						vertexPosition,
-						-vertexPosition,
-						-vertexPosition,
-						vertexPosition,
-						vertexPosition,
-						vertexPosition,
-						-vertexPosition,
-						-vertexPosition,
-						vertexPosition,
-						vertexPosition,
-						-vertexPosition,
-						-vertexPosition,
-						-vertexPosition,
-						-vertexPosition,
-						vertexPosition,
-						-vertexPosition,
-						vertexPosition,
-						-vertexPosition
+						-vertexPosition, -vertexPosition, vertexPosition,
```

Comment:
In long lists here and in the other polyhedra it would be great if a comment were givento` describe what these magic points are. Or maybe label the faces with terms like front, back, left, right, top, bottom. etc.

Reply:
I don't know this too. Cylinders were easy to discover because just there were two groups (and in the old code, the positions of the cube were easy to do without help from three.js).